src/Compilers/CSharp/Test/Emit/Emit/EditAndContinue/EditAndContinueTests.cs

```diff
@@ -16840,5 +16347,788 @@ .maxstack  2
+        [Theory]
+        [InlineData("ComputeStringHash", "string")]
+        [InlineData("ComputeSpanHash", "Span<char>")]
+        [InlineData("ComputeReadOnlySpanHash", "ReadOnlySpan<char>")]
+        public void PrivateImplDetails_ComputeStringHash(string hashMethodName, string typeName)
+        {
+            using var _ = new EditAndContinueTest(targetFramework: TargetFramework.Net80, verification: Verification.Skipped)
+                .AddBaseline(
+                    source: $$"""
+                        using System;
+                        class C
+                        {
+                           static int F({{typeName}} s)
+                               => s switch 
+                               {
+                                   "A_______" => 1,
+                                   "_B______" => 2,
+                                   "__C_____" => 3,
+                                   "___D____" => 4,
+                                   "____E___" => 5,
+                                   "_____F__" => 6,
+                                   "______G_" => 7,
+                                   "_______H" => 8,
+                                   _ => 9
+                               };
+                        }
+                        """,
+                    validator: g =>
+                    {
+                        g.VerifyTypeDefNames("<Module>", "C", "<PrivateImplementationDetails>");
+                        g.VerifyMethodDefNames("F", ".ctor", hashMethodName);
+                    })
+                .AddGeneration(
+                    source: $$"""
+                        using System;
+                        class C
+                        {
+                            static int F({{typeName}} s)
+                               => s switch 
+                               {
+                                   "A_______" => 10,
+                                   "_B______" => 20,
+                                   "__C_____" => 30,
+                                   "___D____" => 40,
+                                   "____E___" => 50,
+                                   "_____F__" => 60,
+                                   "______G_" => 70,
+                                   "_______H" => 80,
+                                   _ => 90
+                               };
+                        }
+                        """,
+                    edits: new[]
+                    {
+                        Edit(SemanticEditKind.Update, symbolProvider: c => c.GetMember("C.F")),
+                    },
+                    validator: g =>
+                    {
+                        g.VerifyTypeDefNames("<PrivateImplementationDetails>#1");
+                        g.VerifyMethodDefNames("F", hashMethodName);
+
+                        g.VerifyEncLogDefinitions(new[]
+                        {
+                            Row(3, TableIndex.StandAloneSig, EditAndContinueOperation.Default),
```

Comment:
How are the baselines constructed for these tests? Are these bits of information written out somewhere or does the test author have to put this together manually?

Reply:
The test prints expected values if it fails. Run it once to get the values, check that they are as expected and copy & paste.